### ISSUE TITLE
Feature request: give examples anchors, to make it possible to link to a specific example

### ISSUE THREAD

**Issue by adrianschmidt** [jgroth/kompendium#60]
In the docs for Lime Elements, we have several components that have a _lot_ of examples, and I regularly find myself wishing I could link straight to an example, rather than just to the docs for the component along with a sentence about scrolling down to example such-and-such.

**Comment by jgroth** [comment]
Yeah! I think this worked at some point :thinking: Can't remember if I just removed that functionality for some reason or if there is a bug
